Rakefile

```
@@ -4,6 +4,7 @@ require 'open-uri'
 require 'shellwords'
 require 'bundler/audit/task'
```

Since you're mucking around here, can we add a blank line to separate official Ruby deps from third party deps?

```suggestion
require 'shellwords'

require 'bundler/audit/task'
```